optgen/cmd/support/agg_gen.go

```diff
@@ -92,7 +92,11 @@ func (g *AggGen) genAggStringer(define AggDef) {
 		sqlName = define.SqlName
 	}
 	fmt.Fprintf(g.w, "func (a *%s)  String() string {\n", define.Name)
-	fmt.Fprintf(g.w, "    return fmt.Sprintf(\"%s(%%s)\", a.Child)\n", strings.ToUpper(sqlName))
+	fmt.Fprintf(g.w, "    res := fmt.Sprintf(\"%s(%%s)\", a.Child)\n", strings.ToUpper(sqlName))
+	fmt.Fprintf(g.w, "    if a.window != nil {\n")
+	fmt.Fprintf(g.w, "    	res = res + \" - \" + a.window.String()\n")
```

Comment:
(minor) You might omit the `-` to be more consistent with other `String()` functions; I don't think we use that format anywhere else.

Reply:
Sure thing. Seems like using space as a separator works just as well, using that to be consistent with other code.